CRM/Admin/Page/APIExplorer.php

```diff
@@ -51,12 +51,25 @@ public function run() {
     $this->assign('operators', CRM_Core_DAO::acceptedSQLOperators());
 
     // List example directories
+    // use get_include_path to ensure that extensions are captured.
     $examples = array();
-    foreach (scandir(\Civi::paths()->getPath('[civicrm.root]/api/v3/examples')) as $item) {
-      if ($item && strpos($item, '.') === FALSE) {
-        $examples[] = $item;
+    $paths = array_unique(explode(PATH_SEPARATOR, get_include_path()));
+    foreach ($paths as $path) {
+      // For some reason there are 2 CiviCRM folder inclusions one with a trailing slash one without
+      // Remove this so that we only get one version of each folder.
+      if (substr($path, -7) == 'civicrm' || substr($path, -17) == 'civicrm//packages') {
+        continue;
+      }
```

Comment:
@seamuslee001 I think this might be different on different sites, and the best way to dedupe would be a little preprocessing to remove trailing slashes before running `array_unique` (note that on some systems the slash will be a backslash)